NUnit Tests/SortModTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using NUnit.Framework;
+using QModManager;
+
+namespace QMMTests
+{
+    [TestFixture]
+    public class SortModTests
+    {
+        [Test]
+        public void Test()
```

Comment:
Sent @ahk1221 some additional unit tests that can be added here.

All this needs is just some more tests to prove out that the code behaves exactly as intended and we're set.